t/12_sles4sap_publicccloud.t

```diff
@@ -112,4 +112,64 @@ subtest '[list_cluster_nodes]' => sub {
     dies_ok { $self->list_cluster_nodes() } 'Expected failure: missing mandatory arg';
 };
 
+subtest '[is_hana_database_offline]' => sub {
+    my $self = sles4sap_publiccloud->new();
+    my $sles4sap_publiccloud = Test::MockModule->new('sles4sap_publiccloud', no_auto => 1);
+    $sles4sap_publiccloud->redefine(get_hana_database_status => sub { return 0; });
+    $sles4sap_publiccloud->redefine(record_info => sub { note(join(' ', 'RECORD_INFO -->', @_)); });
+    set_var('SAP_SIDADM', 'SAP_SIDADMTEST');
+    set_var('INSTANCE_ID', 'INSTANCE_IDTEST');
+    set_var('_HANA_MASTER_PW', '1234');
+
+    my $res = $self->is_hana_database_online();
+    set_var('SAP_SIDADM', undef);
+    set_var('INSTANCE_ID', undef);
+    set_var('_HANA_MASTER_PW', undef);
+    is $res, 0, "Hana database is offline";
+};
+
+subtest '[is_hana_database_offine with status online]' => sub {
```

Comment:
usually we name tests by placing tested function name within square bracket and everything else outside. So it should be 
```
[is_hana_database_offine]  with status online
```